consistency/tool/comparison/main.py

```diff
@@ -0,0 +1,163 @@
+import grpc
+from google.cloud.apigeeregistry.v1 import registry_service_pb2_grpc
+from google.cloud.apigeeregistry.v1 import registry_service_pb2 as rs
+import argparse
+from metrics import vocabulary_pb2
+from comparison import Comparison
+from google.cloud.apigeeregistry.applications.v1alpha1.consistency import (
+    word_group_pb2 as wg,
+)
+from google.cloud.apigeeregistry.v1 import registry_models_pb2 as rm
+
+
+def main():
+    # Creating registry client
+    channel = grpc.insecure_channel("localhost:8080")
+    stub = registry_service_pb2_grpc.RegistryStub(channel)
+
+    parser = argparse.ArgumentParser()
+
+    # add project_name and spec_name commandline arguments.
+    parser.add_argument(
+        "--spec_name",
+        type=str,
+        required=True,
+        help="Name of the spec to compute comparison for",
+    )
+    parser.add_argument(
+        "--project_name",
+        type=str,
+        required=True,
+        help="Name of the project to extract WordGroups and compute comparison for",
+    )
+
+    args = parser.parse_args()
+
+    spec_name = args.spec_name
+    project_name = args.project_name
+    print(spec_name, project_name)
+
+    # Get vocabulary artifacts to generate new spec words
```

Comment:
It might help to add some logging type print statements, like 
"fetching vocabulary artifact"
"fetching wordgroup artifacts"
and so on